atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/InternalSchemaMetadata.java

```diff
@@ -28,9 +31,20 @@
 @JsonSerialize(as = ImmutableInternalSchemaMetadata.class)
 @JsonDeserialize(as = ImmutableInternalSchemaMetadata.class)
 public interface InternalSchemaMetadata {
-    TimestampPartitioningMap<Integer> timestampToTransactionsTableSchemaVersion();
+    // Warning: Do not change this without a migration
+    byte[] DEFAULT_METADATA_COORDINATION_KEY = PtBytes.toBytes("m");
```

Comment:
Put this in `AtlasDbConstants`? Also not sure about the comment, since we have constants like this elsewhere that shouldn't be changed 😢

Reply:
That's fair. I've bumped it to `AtlasDbConstants`.

I think with some of these it's obvious that you can't change them without blowing stuff up (e.g. tablereferences) hence comments aren't needed. In any case I think this one is pretty obviously dangerous too, so removed it.